dhcpv4/dhcpv4.go

```diff
@@ -21,21 +35,20 @@ const MaxMessageSize = 576
 // DHCPv4 represents a DHCPv4 packet header and options. See the New* functions
 // to build DHCPv4 packets.
 type DHCPv4 struct {
-	opcode         OpcodeType
-	hwType         iana.HwTypeType
-	hwAddrLen      uint8
-	hopCount       uint8
-	transactionID  uint32
-	numSeconds     uint16
-	flags          uint16
-	clientIPAddr   net.IP
-	yourIPAddr     net.IP
-	serverIPAddr   net.IP
-	gatewayIPAddr  net.IP
-	clientHwAddr   [16]byte
-	serverHostName [64]byte
-	bootFileName   [128]byte
-	options        []Option
+	OpCode         OpcodeType
```

Comment:
For consistency maybe `Opcode`

Reply:
I'm planning on renaming the types to remove the "Type" suffix as well, because that's kind of unlike Go. And HwTypeType should really be HWType, e.g.